docs/References.bib

```diff
@@ -10,6 +10,23 @@ @article{Arnold2002
   volume  = {39},
 }
 
+@article{Baumgarte1996hh,
```

Comment:
It might also be useful to have a comment at the top (and maybe bottom as well) of the file that the entries should be alphabetized by key; and maybe a comment or link to documentation for any other expectations.  Also should we put the license text in this file?

Reply:
Added a link to the guidelines in the fixup, so that the information on how to format this file is not duplicated